.devcontainer/Dockerfile

```diff
@@ -10,3 +10,12 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
         build-essential python curl git lldb-6.0 liblldb-6.0-dev \
         libunwind8 libunwind8-dev gettext libicu-dev liblttng-ust-dev \
         libssl-dev libnuma-dev libkrb5-dev zlib1g-dev ninja-build
+
+# Install V8 Engine
+SHELL ["/bin/bash", "-c"]
```

Comment:
why is `bash` needed?

Reply:
Without it, I hit this error when using v8 `/usr/local/bin/v8: line 1: 0!/usr/bin/env: No such file or directory`

Reply:
Is that a line ending problem? When I search for that error, the top 2 hits say it is because of line endings.

Reply:
Or is it because you explicitly are putting bash in the script file below?

```
#!/usr/bin/env bash
```

Reply:
Yes, because the usage later in the script.